### ISSUE TITLE
Output results in tool window

### ISSUE THREAD

**Issue by belibug** [nkasco/IT-Admin-Toolkit-WinUI#22]
Issue : Current design launches PowerShell or terminal to run each task. Window automatically closes unless script is configured to wait for input. 

Suggestion : It would be great if the script can run in background and stream output to the tool interface. 

Provide option to choose between launching new PowerShell session or stream to parent window.

**Comment by nkasco** [comment]
I agree, there's some tidbits in the existing code for this but I think there were some limitations b/c you have to launch it in a host powershell process in c#

**Comment by nkasco** [comment]
This should be explored: https://stackoverflow.com/questions/71115270/how-to-host-powershell-5-from-winui-3-c-sharp-app/74317737#74317737
